pkg/cmd/cache_settings/create/create.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/MakeNowJust/heredoc"
 
 	msg "github.com/aziontech/azion-cli/messages/cache_settings"
-	msgapp "github.com/aziontech/azion-cli/messages/edge_applications"
+	// msgapp "github.com/aziontech/azion-cli/messages/edge_applications"
```

Comment:
Hey bro, sorry if i will ask a dumb question, but this shouldn't be removed instead of commented?

Reply:
I left it there for a very simple reason:
Remind myself to redo the messages once this command is ported :D

Reply:
Ahhh gotcha!!!
I have a small suggestion that maybe help you remind these on the future. 
Add a `TODO` with a message that explains the reason, so not only the future Patrick knows the reasons but everyone who looks at the code. 
```suggestion
         TODO: remove the comment when this command is ported
	// msgapp "github.com/aziontech/azion-cli/messages/edge_applications"
```